Source/WebKit/GPUProcess/media/RemoteImageDecoderAVFProxy.cpp

```diff
@@ -88,14 +90,14 @@ void RemoteImageDecoderAVFProxy::deleteDecoder(ImageDecoderIdentifier identifier
 
 void RemoteImageDecoderAVFProxy::encodedDataStatusChanged(ImageDecoderIdentifier identifier)
 {
-    auto connection = m_connectionToWebProcess.get();
+    RefPtr connection = m_connectionToWebProcess.get();
     if (!connection)
         return;
     if (!m_imageDecoders.contains(identifier))
         return;
 
-    auto imageDecoder = m_imageDecoders.get(identifier);
-    connection->connection().send(Messages::RemoteImageDecoderAVFManager::EncodedDataStatusChanged(identifier, imageDecoder->frameCount(), imageDecoder->size(), imageDecoder->hasTrack()), 0);
+    RefPtr imageDecoder = m_imageDecoders.get(identifier);
```

Comment:
Instead of doing contains and get, can we just do get and then `if (!imageDecoder) return;`?

Reply:
That is much cleaner--Fixed!